src/test/java/api/support/builders/LoanPolicyBuilder.java

```diff
@@ -322,6 +322,31 @@ public LoanPolicyBuilder withLoanable(boolean loanable) {
       this.gracePeriod);
   }
 
+  public LoanPolicyBuilder withLoanPeriod(JsonObject loanPeriod) {
+    return new LoanPolicyBuilder(
+      this.id,
+      this.name,
+      this.description,
+      this.loanable,
+      this.profile,
+      loanPeriod,
+      this.fixedDueDateScheduleId,
+      this.closedLibraryDueDateManagementId,
+      this.openingTimeOffsetPeriod, this.unlimitedRenewals,
```

Comment:
Please either use a parameter per line or use the standard line separation

Reply:
Removed this method altogether - not needed after using the rolling loan policy.